### PR TITLE
Added support for RGB on the Tada68

### PR DESCRIPTION
**Pull request by Maartenwut**

### CODE REVIEW

**Comment by jackhumbert** [comment]
Cool! Since this isn't available without modification, let's keep the default as it is, but have another keymap that can be used as an example :) `rgb` would be fine, or if you wanna just use your keymap, that works too!

**Comment by Maartenwut** [comment]
Done!

File: keyboards/tada68/keymaps/default/keymap.c [hunk 1]
```diff
@@ -38,14 +38,14 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
...
-   * |        |   |   |Bl-|BL |BL+|   |VU-|VU+|MUT|   |MouseL|MsU|Rck |
+   * |        |   |   |Bl-|BL |BL+|   |VU-|VU+|MUT|   |   McL|MsU|McR |
    * |----------------------------------------------------------------|
    * |    |    |    |                       |   |   |    |MsL|MsD|MsR |
    * `----------------------------------------------------------------'
    */
 [_FL] = KEYMAP_ANSI(
   _______, KC_F1 ,KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_DEL, KC_INS ,  \
-  _______,_______,KC_UP, _______,_______,_______,_______,_______,_______,_______,_______, BL_DEC,BL_INC, BL_TOGG,KC_HOME, \
+  _______,_______,_______,_______, KC_UP, _______,_______,_______,_______,_______,_______,_______,_______, _______,KC_HOME, \
```
Comment (jackhumbert) [inline keyboards/tada68/keymaps/default/keymap.c]: Is there any reason for this? It looks like `KC_UP` may be misaligned now. This still has a backlight, right?

File: keyboards/tada68/config.h [hunk 2]
```diff
@@ -66,15 +66,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
 #define RGBLED_NUM 2     // Number of LEDs
-#define RGBLIGHT_HUE_STEP 10
-#define RGBLIGHT_SAT_STEP 17
-#define RGBLIGHT_VAL_STEP 17*/
-//Disabled while I figure out a suitable pin for RGB support.
-//I've tried F5, D2, D3, and E2 but it's possible the end of my
-//strand is bad. New LEDs on order.
+#define RGBLIGHT_HUE_STEP 8
+#define RGBLIGHT_SAT_STEP 8
+#define RGBLIGHT_VAL_STEP 8
```
Comment (jackhumbert) [inline keyboards/tada68/config.h]: Would you mind moving this to the keymap's config as well? It'll change per keymap :)

**Comment by Maartenwut** [comment]
Woops, made a mistake there. Also moved the RGB config to the keymap.

**Comment by jackhumbert** [comment]
Perfect :) thanks!

**Pull request merged**